pkg/events/registry.go

```diff
@@ -0,0 +1,37 @@
+package events
+
+import (
+	"k8s.io/klog"
+)
+
+type EventRegistry struct {
+	eventHandlers map[string]EventHandler
```

Comment:
Can we assume handlers will be registered on startup by the same thread so we don't have to worry about synchronization?

Reply:
Yes, that's the plan, but very good point. May be it's a fact that we may need to document.